src/core/qgsdistancearea.cpp

```diff
@@ -537,6 +540,111 @@ double QgsDistanceArea::measureLine( const QgsPoint &p1, const QgsPoint &p2, Qgs
   return result;
 }
 
+double QgsDistanceArea::measureLineProjected( const QgsPoint &p1, double distance, double azimuth, QgsPoint *projected_point )
+{
+  double result = 0.0;
+  QgsPoint p2;
+  if ( geographic() )
```

Comment:
This has been removed and the definition set to const
* python-script adapted

Waiting now for everything to compile and (re-) the script, before committing.